addons/web/static/src/core/py_js/py_interpreter.js

```diff
@@ -352,6 +352,8 @@ const allowedFns = new Set([
     ...Object.values(SET),
     ...Object.values(DICT),
     ...Object.values(STRING),
+    Math.min,
+    Math.max,
```

Comment:
if you add min and max to the py_interpreter code, you should also add the support for it. Just adding it to the safe list is half the work. Could you add the min and max function to the builtins, so users will not need to add it to the eval context? 
Also, you should simply write a new builtin max function that simply call Math.max with args, ignoring kwargs, so you don't need your change below. Same for min